core/arch/arm/mm/tee_mmu.c

```diff
@@ -262,33 +359,74 @@ static void asid_free(unsigned int asid)
 	cpu_spin_unlock_xrestore(&g_asid_spinlock, exceptions);
 }
 
-TEE_Result tee_mmu_init(struct user_ta_ctx *utc)
+static TEE_Result map_kinit(struct user_ta_ctx *utc __maybe_unused)
 {
+#ifdef CFG_CORE_UNMAP_CORE_AT_EL0
+	struct mobj *mobj;
+	size_t offs;
+	vaddr_t va;
```

Comment:
`vm_map` expects argument &va to point to an initialized value.

Reply:
Yes, it's initialized by the call to `thread_get_user_kcode()`

Reply:
right, my fault.